atlasdb-api/src/main/java/com/palantir/atlasdb/config/LeaderConfig.java

```diff
@@ -47,15 +49,13 @@ public File acceptorLogDir() {
     public abstract String localServer();
 
     @Size(min = 1)
-    public abstract Set<String> leaders();
+    public abstract List<String> leaders();
```

Comment:
why can't we leave `leaders` as a set, then selecting the lockCreator is just sorting the set and selecting the first:

```
    public String lockCreator() {
        return leaders().stream().sorted().findFirst()
                .orElseThrow(() -> new IllegalStateException("The leaders block cannot be empty"));
    }
```